pkg/minikube/node/start.go

```diff
@@ -90,11 +90,13 @@ type Starter struct {
 
 // Start spins up a guest and starts the Kubernetes node.
 func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
+	var wg sync.WaitGroup
 	stop8ks, err := handleNoKubernetes(starter)
 	if err != nil {
 		return nil, err
 	}
 	if stop8ks {
+		configureMounts(&wg, *starter.Cfg)
```

Comment:
when kubernetes actually starts, this only gets run if the driver is VM-based. do we need to check for that here too?

Reply:
Good call, that slipped my mind, I moved the check within `configureMounts` to simplify